dmoj/cli.py

```diff
@@ -330,23 +331,19 @@ def execute(self):
         print highlight(file_diff, guess_lexer(file_diff), terminal256())
 
 class ShowSubmissonIdOrFilename(Command):
-    name: 'show'
-    help: 'Shows file based on submission ID or filename'
+    name = 'show'
+    help = 'Shows file based on submission ID or filename'
 
     def _populate_parser(self):
         self.arg_parser.add_argument('submission_id', type=int, help='id of submission to show')
```

Comment:
Should have only one parameter, `id_or_source`, and assume it's a submission id if it's an `int`, source file if it's not.